src/PKSim.Core/Model/CompoundPK.cs

```diff
@@ -1,37 +1,89 @@
-﻿using OSPSuite.Utility.Collections;
+using System.Linq;
+using OSPSuite.Core.Domain;
+using OSPSuite.Utility.Collections;
 
 namespace PKSim.Core.Model
 {
    /// <summary>
-   /// Reperesents a cache containing specific PK-Values calculated for a given compound
+   /// Represents a cache containing specific PK-Values calculated for a given compound
    /// </summary>
    public class CompoundPK
```

Comment:
This object is the basis for the context. We are collecting denominotors for ratio parameters as well as QuantityPKParameters where necessary. Sometimes the context is that we have the parameter already calculated, sometimes we have only the denominator